frontend/src/views/SignUp.vue

```diff
@@ -64,21 +101,43 @@
     </v-container>
   </v-sheet>
 </template>
-
 <script>
 export default {
   name: 'SignUp',
-
   data() {
     return {
       email: null,
       password: null,
       user: null,
       progress: false,
+      name: null,
+      bio: null,
+      website: null,
+      rules: [
+        value => !!value || 'Required.',
+        value => value.trim().length > 0 || 'Enter Valid Name or Username'],

```

Comment:
Put these 2 together:

`value => (values && value.trim().length > 0) || 'Field is Required'`